TheFourFifths/consus-client
# Timeout TFF-80

### Summary

Added the auto checkout on timeout functionality. The system will now automatically attempt to checkout the items in the cart after 1 minute.

### Checklist


Finally, is it ready to be reviewed and merged: sure :green_apple: 

### How to Review

1. Pull
2. Input Student ID
3. Add Item to Cart. 
4. Wait 1 minute.
5. Item should be checked out.

### Links

Put links to Jira tickets, and anything else, here.

- [TFF-80](https://msoese.atlassian.net/browse/TFF-80)
